source/models/customizing-serializers.md

```diff
@@ -135,7 +136,40 @@ import DS from 'ember-data';
 export default DS.JSONSerializer.extend({});
 ```
 
-#### ID
+To change the format of the data that is sent to the backend store, you can use
+the `serialize` hook:
+
+```app/serializers/application.js
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+  serialize(snapshot, options) {
+    // help! what do i do here?
```

Comment:
Help! What do I do here? Can someone provide a code example similar to the one below for normalizing?